app/models/concerns/annotating.rb

```diff
@@ -8,6 +8,8 @@ def annotated
       class_eval do
         include Model
         include ManagedBy
+
+        background_deletion << :annotations
```

Comment:
Maybe you can do this and don't remove the `freeze`?

```suggestion
        background_deletion = [*background_deletion, :annotations]
```